xml/obs_best_practice_staging_workflow.xml

```diff
@@ -122,7 +122,7 @@
       <listitem>
         <para>
           <emphasis>Obsolete Requests:</emphasis>
-          Requests that was declined, revoked or superseded.
+          Requests that were declined, revoked or superseded.
```

Comment:
@dmarcoux not 100% sure, but i think it need to be "were"

Reply:
Yes